tms_product/models/product_template.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2024 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    is_vehicle = fields.Boolean(string="Is a Vehicle")
+    vehicle_type = fields.Selection(selection=[("car", "Car"), ("bike", "Bike")])
```

Comment:
@EdgarRetes I believe you don't need to redefine the list of options here. This field should be a related field.